Exercises.Linq/TaskData.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Collections.Generic;
+
+namespace Exercises.Linq
+{
+    public class TaskData
+    {
+        public List<string> Fruits { get; } = new List<string>()
+            {
+                "Lemon", "Apple", "Orange", "Lime", "Watermelon", "Loganberry"
+            };
+
+        public List<int> MixedNumbers { get; } = new List<int>() { 15, 8, 21, 24, 32, 13, 30, 12, 7, 54, 48, 4, 49, 96 };
```

Comment:
Strive to use interfaces with minimal number of members (`ICollection` or `IEnumerable`)